[2022-02-02 21:39:38,803] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Question4_ZONE.dowdata manual__2022-02-02T21:39:36.511344+00:00 [queued]>
[2022-02-02 21:39:38,870] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Question4_ZONE.dowdata manual__2022-02-02T21:39:36.511344+00:00 [queued]>
[2022-02-02 21:39:38,871] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 21:39:38,871] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-02 21:39:38,871] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 21:39:38,929] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): dowdata> on 2022-02-02 21:39:36.511344+00:00
[2022-02-02 21:39:38,938] {standard_task_runner.py:52} INFO - Started process 1976 to run task
[2022-02-02 21:39:38,943] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'Question4_ZONE', 'dowdata', 'manual__2022-02-02T21:39:36.511344+00:00', '--job-id', '362', '--raw', '--subdir', 'DAGS_FOLDER/dag_questions4.py', '--cfg-path', '/tmp/tmplt_jq4_q', '--error-file', '/tmp/tmpimc3s5g7']
[2022-02-02 21:39:38,944] {standard_task_runner.py:77} INFO - Job 362: Subtask dowdata
[2022-02-02 21:39:39,169] {logging_mixin.py:109} INFO - Running <TaskInstance: Question4_ZONE.dowdata manual__2022-02-02T21:39:36.511344+00:00 [running]> on host d574c7c5b085
[2022-02-02 21:39:39,535] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Question4_ZONE
AIRFLOW_CTX_TASK_ID=dowdata
AIRFLOW_CTX_EXECUTION_DATE=2022-02-02T21:39:36.511344+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-02T21:39:36.511344+00:00
[2022-02-02 21:39:39,539] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-02 21:39:39,549] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sSLf https://s3.amazonaws.com/nyc-tlc/misc/taxi+_zone_lookup.csv > /opt/***taxi+_zone_lookup.csv']
[2022-02-02 21:39:39,569] {subprocess.py:85} INFO - Output:
[2022-02-02 21:39:40,997] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-02 21:39:41,158] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=Question4_ZONE, task_id=dowdata, execution_date=20220202T213936, start_date=20220202T213938, end_date=20220202T213941
[2022-02-02 21:39:41,227] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-02 21:39:41,408] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
